libs/libnetconf2/patches/001-cmake_not_updated.patch

```diff
@@ -1,11 +1,16 @@
---- a/CMakeLists.txt	2020-05-08 13:29:30.903800624 +0300
-+++ b/CMakeLists.txt	2020-05-08 13:29:50.943582218 +0300
-@@ -153,7 +153,7 @@
-     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DNC_ENABLED_SSH")
-     message(STATUS "LibSSH version ${LibSSH_VERSION} found")
- 
--    if(LibSSH_VERSION VERSION_EQUAL 0.9.3 OR LibSSH_VERSION VERSION_EQUAL 0.9.4)
-+    if(LibSSH_VERSION VERSION_EQUAL 0.9.x)
-         message(FATAL_ERROR "LibSSH ${LibSSH_VERSION} includes regression bugs and libnetconf2 will NOT work properly, try to use another version")
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f21fec9..5b912af 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -148,7 +148,7 @@ endif()
+ # dependencies - libssh
+ if(ENABLE_SSH)
+     find_package(LibSSH 0.7.0 REQUIRED)
+-    if(LIBSSH_VERSION VERSION_EQUAL 0.9.3 OR LIBSSH_VERSION VERSION_EQUAL 0.9.4)
```

Comment:
there are problems with both 0.9.3 and 0.9.4, I am using libssh0.9.2 now, is that ok?

Reply:
Yes.
I found the patch for libssh 0.9.4 that should solve the compatilibity problem, so I will add that to libssh, that should make 0.9.4 version usable with libnetconf2

Reply:
Yes, I do now, I will test the patch I found for this